### DISCUSSION TITLE
Become a maintainer

### DISCUSSION THREAD

**Discussion by asturio** [LibrePDF/OpenPDF#1089] [Announcements]
Here an open "discussion" for people wanting to become a co-maintainer of OpenPDF, and wanting to help out in a regular basis with:

- Pull request reviewing
- Pull request merging
- Issues
- Discussions

To be sure, that I read your reply, mention me with a "@asturio".

---
### Update 2024-04-04
After the xz-Tools drama, I'm rethinking a how open the project is for new maintainers, so I'm going a new strategy for finding 1 new maintainer to help me out with all the management of OpenPDF.

OpenPDF won't be here without all the contributors it has. My very big "thank you" for all of you. I'm also very thankful for the previous maintainers of this great project.

I'll take a look at the git-history of OpenPDF and try to find some candidates to help me:

- Reviewing and merging pull request
- Participating by Issues
- Participating by Discussions
- And maybe also releasing new versions of this library

**Comment by ansidev** [comment]
@asturio How to become a co-maintainer?

**Reply by asturio** [reply]
To **contribute** you only need a GitHub-account. As a contributor, you can:

- Write and comment in the Issues
- Participate in the Discussions
- Contribute with some Pull-Requests (fixing bugs or adding new features)

As a maintainer, you should also review some pull-requests and merge them, if they are ok.

Just start contributing, in the project, and if I think your contributions are nice, I'll gladly add you as a maintainer.

Here is a nice article on how to contribute to open-source-software: https://github.com/readme/guides/first-oss-contribution

**Comment by asturio** [comment]
I'll mention candidates here.
